mne/io/base.py

```diff
@@ -1923,6 +1923,30 @@ def describe(self, data_frame=False):
             msg += f"{cols['max'][i]:>8.2f}"
             print(msg)
 
+    @verbose
+    def rename_annotations(self, mapping, verbose=None):
+        """Rename annotation description(s). Operates inplace.
+
+        Parameters
+        ----------
+        mapping : dict
+            A dictionary mapping the old description to a new description
+            name e.g. {'1.0' : 'Control', '2.0' : 'Stimulus'}.
+        %(verbose_meth)s
+
+        Returns
+        -------
+        self : mne.Annotations
+            The modified Annotations object.
```

Comment:
Right now this is inaccurate; it will return the Raw instance, not the annotations instance.  And in fact the raw method *should* return the raw instance IMO, so the docstring should be updated.